src/app/rpmostree-builtin-status.cxx

```diff
@@ -272,6 +273,24 @@ get_last_auto_update_run (GDBusConnection   *connection,
   return TRUE;
 }
 
+static gboolean
+get_update_driver_sd_unit (char **update_driver, GError **error)
+{ 
+  if (!g_file_test (RPMOSTREE_DRIVER_STATE, G_FILE_TEST_EXISTS))
+    return TRUE;
+  g_autoptr(GHashTable) driver_info = g_hash_table_new_full (g_str_hash, g_str_equal, g_free,
+                                                   (GDestroyNotify)g_variant_unref);
+  if (!rpmostree_composeutil_read_json_metadata_from_file (RPMOSTREE_DRIVER_STATE, driver_info, error))
```

Comment:
Would it make sense to move this function from `rpmostree-composeutil.cxx` to somewhere less specific like `rpmostree-libbuiltin` or `libpriv/rpmostree-json-parsing.cxx`?